MIVisionX-setup.py

```diff
@@ -222,9 +222,8 @@
 			os.system('sudo -v')
 			os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install libboost-all-dev clang')
 			#turbo-JPEG
-			os.system('(cd '+deps_dir+'; wget https://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-2.0.3.tar.gz )')
-			os.system('(cd '+deps_dir+'; tar xf libjpeg-turbo-2.0.3.tar.gz )')
-			os.system('(cd '+deps_dir+'/libjpeg-turbo-2.0.3; mkdir build; cd build; '+linuxCMake+' -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=RELEASE -DENABLE_STATIC=FALSE -DCMAKE_INSTALL_DOCDIR=/usr/share/doc/libjpeg-turbo-2.0.3 -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib ..; make -j 4; sudo make install )')
+			os.system('(cd '+deps_dir+'; git clone https://github.com/Indumathi31/libjpeg-turbo.git )')
```

Comment:
can we get a release or something constant? Master branch can be updated.

Reply:
aby update on this @LakshmiKumar23 ?

Reply:
@kiritigowda Spoke to MCW about this today. They will give me us a release version in a day

Reply:
@kiritigowda changed to release version 2.0.4. Please verify